docker/archimedes-base/Dockerfile

```diff
@@ -9,7 +12,7 @@ ENV DOCKERIZE_URL https://github.com/jwilder/dockerize/releases/download/v${DOCK
 ENV PATH "/app/bin:/opt/poetry/bin:$PATH"
 ENV POETRY_HOME "/opt/poetry"
 
-COPY certs/rootCA.pem /usr/local/share/ca-certificates/devRoot.crt
+COPY --from=1 /certs/rootCA.pem /usr/local/share/ca-certificates/devRoot.crt
```

Comment:
BOOM!  Before, I had to do inception like magic to 'mount' other project directories into the build context, but now we can simply copy the certs from the previously built other image.

`--from=1` refers to the 2nd (0 index) `FROM` statement in the multistage build.  I'm copy certs from the `development-certs` image.